tests/bootstrap.php

```diff
@@ -108,6 +110,8 @@
     ConnectionManager::get('test')->getDriver()->enableAutoQuoting(true);
 }
 
+FactoryLocator::add('Table', new TableLocator());
```

Comment:
We could add this to a new bootstrap file in the ORM package (autoloaded by compose) instead for friction free usage/upgrade.

Reply:
I think that will make our lives and our user's lives more simple. We would need the bootstrap in both the `cakephp/cakephp` and `cakephp/orm` packages.

Reply:
Done